template/package.json.ejs

```diff
@@ -6,10 +6,10 @@
   "cacheDirectories": [".next/cache"],
 <% } -%>
   "scripts": {
-    "build": "yarn build:nexus && yarn build:prisma && yarn build:next",
+    "build": "ts-node ./scripts/buildProd",
     "build:prisma": "prisma generate",
     "build:next": "next build",
-    "build:nexus": "ts-node -P tsconfig.cjs.json --transpile-only graphql/schema.ts",
+    "build:nexus": "NODE_ENV=development ts-node -P tsconfig.cjs.json --transpile-only graphql/schema.ts",
```

Comment:
This is one of the biggest gotchas I've ever seen. Nexus will not generate types if `NODE_ENV=production`. It's not documented well, but I did eventually find one place where it was:

![image](https://user-images.githubusercontent.com/14339/96814665-d878c300-13ea-11eb-9279-5c779bd76da4.png)

This bug came about after #68

Reply:
I'm so glad you figured this one out! A great reason for us to be running the tests of the `created app`